<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>LoginWindow</class>
	<widget class="QWidget" name="LoginWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1024</width>
				<height>768</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>CSE</string>
		</property>
		<property name="styleSheet">
			<string notr="true">
				/* General Background */
				QWidget#LoginWindow {
				background-color: #f0f4f8;
				}

				/* Header and Footer */
				QFrame#headerFrame, QFrame#footerFrame {
				background-color: #0c3b6d;
				border: none;
				}

				/* Header Text */
				QLabel#logoLabel {
				color: white;
				font-size: 20px;
				font-weight: bold;
				}
				
				/* Footer Text */
				QLabel#footerLinks {
				color: white;
				}

				/* Login Card Container */
				QFrame#loginCard {
				background-color: white;
				border: 1px solid #dcdcdc;
				border-top: 4px solid #0c3b6d;
				border-radius: 4px;
				}

				/* Inputs */
				QLineEdit {
				border: 1px solid #cccccc;
				border-radius: 4px;
				padding: 8px;
				font-size: 14px;
				color: #333;
				}
				QLineEdit:focus {
				border: 1px solid #0c3b6d;
				}

				/* Labels */
				QLabel {
				color: #555;
				font-size: 13px;
				}

				/* Primary Button (Log In) */
				QPushButton#loginBtn {
				background-color: #0c3b6d;
				color: white;
				border-radius: 4px;
				padding: 10px;
				font-weight: bold;
				font-size: 14px;
				}
				QPushButton#loginBtn:hover {
				background-color: #0a2e55;
				}

				/* Secondary Button (Register) - Styled to look clean */
				QPushButton#registerBtn {
				background-color: transparent;
				color: #0c3b6d;
				border: none;
				font-weight: bold;
				font-size: 13px;
				text-align: center;
				}
				QPushButton#registerBtn:hover {
				text-decoration: underline;
				}

				/* Links (Forgot Password) */
				QLabel#forgotPassLabel {
				color: #0c3b6d;
				}

				/* 2FA Warning Box */
				QFrame#warningFrame {
				background-color: #f1f4f6;
				border-top: 1px solid #e0e0e0;
				border-bottom-left-radius: 4px;
				border-bottom-right-radius: 4px;
				}
			</string>
		</property>
		<layout class="QVBoxLayout" name="verticalLayout">
			<property name="spacing">
				<number>0</number>
			</property>
			<property name="leftMargin">
				<number>0</number>
			</property>
			<property name="topMargin">
				<number>0</number>
			</property>
			<property name="rightMargin">
				<number>0</number>
			</property>
			<property name="bottomMargin">
				<number>0</number>
			</property>
			<item>
				<widget class="QFrame" name="headerFrame">
					<property name="minimumSize">
						<size>
							<width>0</width>
							<height>70</height>
						</size>
					</property>
					<layout class="QHBoxLayout" name="horizontalLayout_2">
						<property name="leftMargin">
							<number>40</number>
						</property>
						<item>
							<widget class="QLabel" name="logoLabel">
								<property name="text">
									<string>CSE</string>
								</property>
							</widget>
						</item>
						<item>
							<spacer name="horizontalSpacer">
								<property name="orientation">
									<enum>Qt::Horizontal</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>40</width>
										<height>20</height>
									</size>
								</property>
							</spacer>
						</item>
					</layout>
				</widget>
			</item>

			<item>
				<widget class="QWidget" name="contentArea" native="true">
					<layout class="QVBoxLayout" name="verticalLayout_2">
						<item>
							<spacer name="verticalSpacer_1">
								<property name="orientation">
									<enum>Qt::Vertical</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>20</width>
										<height>40</height>
									</size>
								</property>
							</spacer>
						</item>

						<item>
							<layout class="QHBoxLayout" name="horizontalLayout_3">
								<item>
									<spacer name="horizontalSpacer_2">
										<property name="orientation">
											<enum>Qt::Horizontal</enum>
										</property>
										<property name="sizeHint" stdset="0">
											<size>
												<width>40</width>
												<height>20</height>
											</size>
										</property>
									</spacer>
								</item>
								<item>
									<widget class="QFrame" name="loginCard">
										<property name="minimumSize">
											<size>
												<width>380</width>
												<height>420</height>
											</size>
										</property>
										<property name="maximumSize">
											<size>
												<width>380</width>
												<height>450</height>
											</size>
										</property>
										<layout class="QVBoxLayout" name="verticalLayout_3">
											<property name="spacing">
												<number>15</number>
											</property>
											<property name="leftMargin">
												<number>0</number>
											</property>
											<property name="topMargin">
												<number>30</number>
											</property>
											<property name="rightMargin">
												<number>0</number>
											</property>
											<property name="bottomMargin">
												<number>0</number>
											</property>

											<item>
												<layout class="QVBoxLayout" name="cardContent">
													<property name="leftMargin">
														<number>30</number>
													</property>
													<property name="rightMargin">
														<number>30</number>
													</property>

													<item>
														<widget class="QLabel" name="labelUser">
															<property name="text">
																<string>Username/User ID</string>
															</property>
														</widget>
													</item>
													<item>
					<widget class="QLineEdit" name="userEdit">
						<property name="styleSheet">
							<string>color: white;</string>
						</property>
					</widget>
				</item>
													<item>
														<widget class="QLabel" name="labelPass">
															<property name="text">
																<string>Password</string>
															</property>
														</widget>
													</item>
													<!-- Replacing the passEdit widget item with a layout that includes passEdit and showPassBtn -->
													<item>
														<layout class="QHBoxLayout" name="passwordLayout">
															<item>
							<widget class="QLineEdit" name="passEdit">
								<property name="echoMode">
									<enum>QLineEdit::Password</enum>
								</property>
								<property name="styleSheet">
									<string>color: white;</string>
								</property>
							</widget>
						</item>
															<item>
																<widget class="QPushButton" name="showPassBtn">
																	<property name="text">
																		<string>üëÅÔ∏è</string>
																	</property>
																	<property name="checkable">
																		<bool>true</bool>
																	</property>
																	<property name="cursor">
																		<cursorShape>PointingHandCursor</cursorShape>
																	</property>
																</widget>
															</item>
														</layout>
													</item>
													<item>
														<widget class="QLabel" name="forgotPassLabel">
															<property name="text">
																<string>Forgot Password?</string>
															</property>
															<property name="alignment">
																<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
															</property>
														</widget>
													</item>
													<item>
														<widget class="QPushButton" name="loginBtn">
															<property name="text">
																<string>Log In</string>
															</property>
															<property name="cursor">
																<cursorShape>PointingHandCursor</cursorShape>
															</property>
														</widget>
													</item>

													<item>
														<widget class="QPushButton" name="registerBtn">
															<property name="text">
																<string>Register for Online Banking</string>
															</property>
															<property name="cursor">
																<cursorShape>PointingHandCursor</cursorShape>
															</property>
															<property name="flat">
																<bool>true</bool>
															</property>
														</widget>
													</item>

												</layout>
											</item>

											<item>
												<spacer name="verticalSpacer_3">
													<property name="orientation">
														<enum>Qt::Vertical</enum>
													</property>
													<property name="sizeHint" stdset="0">
														<size>
															<width>20</width>
															<height>40</height>
														</size>
													</property>
												</spacer>
											</item>

											<item>
												<widget class="QFrame" name="warningFrame">
													<property name="minimumSize">
														<size>
															<width>0</width>
															<height>70</height>
														</size>
													</property>
													<layout class="QHBoxLayout" name="horizontalLayout_4">
														<item>
															<widget class="QLabel" name="lockIcon">
																<property name="maximumSize">
																	<size>
																		<width>30</width>
																		<height>30</height>
																	</size>
																</property>
																<property name="text">
																	<string>üîí</string>
																</property>
																<property name="alignment">
																	<set>Qt::AlignCenter</set>
																</property>
															</widget>
														</item>
														<item>
															<widget class="QLabel" name="warningText">
																<property name="text">
																	<string>Two-factor authentication is required for security.</string>
																</property>
																<property name="wordWrap">
																	<bool>true</bool>
																</property>
															</widget>
														</item>
													</layout>
												</widget>
											</item>
										</layout>
									</widget>
								</item>
								<item>
									<spacer name="horizontalSpacer_3">
										<property name="orientation">
											<enum>Qt::Horizontal</enum>
										</property>
										<property name="sizeHint" stdset="0">
											<size>
												<width>40</width>
												<height>20</height>
											</size>
										</property>
									</spacer>
								</item>
							</layout>
						</item>

						<item>
							<spacer name="verticalSpacer_2">
								<property name="orientation">
									<enum>Qt::Vertical</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>20</width>
										<height>40</height>
									</size>
								</property>
							</spacer>
						</item>
					</layout>
				</widget>
			</item>

			<item>
				<widget class="QFrame" name="footerFrame">
					<property name="minimumSize">
						<size>
							<width>0</width>
							<height>50</height>
						</size>
					</property>
					<layout class="QHBoxLayout" name="horizontalLayout">
						<property name="leftMargin">
							<number>40</number>
						</property>
						<property name="rightMargin">
							<number>40</number>
						</property>
						<item>
							<widget class="QLabel" name="badgesLabel">
								<property name="text">
									<string>üõ°Ô∏è Security Guaranteed</string>
								</property>
								<property name="styleSheet">
									<string>color: white;</string>
								</property>
							</widget>
						</item>
						<item>
							<spacer name="horizontalSpacer_4">
								<property name="orientation">
									<enum>Qt::Horizontal</enum>
								</property>
								<property name="sizeHint" stdset="0">
									<size>
										<width>40</width>
										<height>20</height>
									</size>
								</property>
							</spacer>
						</item>
						<item>
							<widget class="QLabel" name="footerLinks">
								<property name="text">
									<string>Privacy Policy    Contact Us</string>
								</property>
								<property name="alignment">
									<set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
								</property>
							</widget>
						</item>
					</layout>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>